Entities/MOPixel.cpp

```diff
@@ -196,10 +196,15 @@ namespace RTE {
 			if (m_DistanceTraveled > m_LethalRange) {
 				if (m_Sharpness < m_LethalSharpness) {
 					m_Sharpness = std::max(m_Sharpness * (1.0F - (20.0F * g_TimerMan.GetDeltaTimeSecs())) - 0.1F, 0.0F);
+					if (m_LethalRange > 0) {
```

Comment:
This part just adds a bit of variance to how LethalRange affects bullets - the TrailLength is shortened to give bullets a slight fade-out-of-existence effect, while RNG determines exactly how fast they render unlethal. The moment the bullet no longer `HitsMOs`, the starting conditions of this function are no longer fulfilled.

Reply:
Just noticed this was still around. I guess I hadn't noticed the hitsMOs check above so that makes sense.
Please do try to update the variable comment in the header to better match its new reality though, right now it's `//!< After this distance in meters, the damage output of this MO decrease. Default is half a screen plus sharp length for pixels spawned from a gun and one screen otherwise.`